repose-aggregator/tests/release-verification/src/docker/deb/Dockerfile

```diff
@@ -0,0 +1,51 @@
+# To build this image, use a command like the following:
+# docker build --build-arg REPOSE_VERSION=latest -t repose-container .
+# To run the built image, use a command like the following:
+# docker run --rm repose-container
+FROM ubuntu:xenial
+
+MAINTAINER The Repose Team <reposecore@rackspace.com>
+
+# This build-arg is used to pass the Repose version number which will be set up in this image.
+ARG REPOSE_VERSION
+
+# The service ports -- one for accepting HTTP traffic to Repose, and the other for connecting a JDWP debugger
+# Note that these are not exposed to the host by default, but can be using the Docker CLI -p or -P flag.
+EXPOSE 8080 10037
+
+RUN apt-get update -qq && apt-get install -y -qq \
+    wget \
+    curl \
+    python-pip
+RUN wget --quiet http://mirrors.kernel.org/ubuntu/pool/universe/r/rlwrap/rlwrap_0.41-1build1_amd64.deb
+RUN dpkg -i rlwrap_0.41-1build1_amd64.deb
+RUN wget --quiet https://deb.nodesource.com/node_0.12/pool/main/n/nodejs/nodejs_0.12.7-1nodesource1~trusty1_amd64.deb
+RUN dpkg -i nodejs_0.12.7-1nodesource1~trusty1_amd64.deb
+RUN apt-mark hold nodejs
+RUN apt-get install -y -f
+RUN wget -O - http://repo.openrepose.org/debian/pubkey.gpg | apt-key add -
+RUN echo "deb http://repo.openrepose.org/debian stable main" > /etc/apt/sources.list.d/openrepose.list
+
+# Note: The COPY instruction was used rather than VOLUME since the behave of VOLUME varies based on the host OS
+COPY etc_repose /release-verification/etc_repose
+COPY fake-services /release-verification/fake-services
+COPY scripts /release-verification/scripts
+COPY *.deb /release-verification/
+
+RUN rm /release-verification/DELETE-ME.deb
+
+RUN sh /release-verification/scripts/fake_keystone_prepare.sh
+RUN cp /release-verification/fake-services/fake-keystone2/package.json /opt/fake-keystone/package.json
+RUN cp /release-verification/fake-services/fake-keystone2/app.js /opt/fake-keystone/app.js
+RUN sh /release-verification/scripts/docker_fake_keystone_install.sh
```

Comment:
The Vagrant scripts invoked `sudo` since Vagrant ssh's as a non-root user. In Docker, I was just doing everything as the root user since it's a container, and invoking `sudo` as the root user was causing `command not found` errors.